app/models/user.rb

```diff
@@ -325,7 +326,7 @@ def tech_admin?
   end
 
   def pro?
-    has_role?(:pro)
+    pro_membership&.active? || has_role?(:pro)
```

Comment:
We change the meaning of `pro?` so that user's with active memberships are automatically pro. The role can still be used for existing beta pro users or for whomever the Pro status is gifted